src/App.tsx

```diff
@@ -653,6 +655,25 @@ function AppInner() {
                   }
                 />
               </ErrorBoundary>
+            ) : projects.size === 1 ? (
```

Comment:
Don't we wanna show this as the default path when creating a new workspace in a project? Like, even when you have existing workspaces / projects?